app/Handlers/Utility/CalculateHandler.php

```diff
@@ -23,6 +23,7 @@
     private $calculator;
 
     /**
+     * @param JsonLoader $loader
```

Comment:
This is fixed in a different PR, so please reset this to keep features isolated in commits.